contracts/MultiCreateV1.sol

```diff
@@ -0,0 +1,105 @@
+pragma solidity ^0.8.7;
+import "./interfaces/IControllerV1.sol";
+import "./interfaces/IMemberToken.sol";
+
+/* solhint-disable indent */
+
+contract MultiCreateV1 {
+    IMemberToken immutable memberToken;
+
+    constructor(address _memberToken) {
+        memberToken = IMemberToken(_memberToken);
+    }
+
+    struct AdminPointer {
+        uint256 podId;
+        address pointer;
+    }
+
+    function createPods(
+        IControllerV1 _controller,
+        address[][] memory _members,
+        uint256[] calldata _thresholds,
+        address[] memory _admins,
+        bytes32[] memory _labels,
+        string[] memory _ensStrings,
+        string[] memory _imageUrls
+    ) public returns (address[] memory) {
+        uint256 numPods = _thresholds.length;
+        require(_members.length == numPods, "incorrect members array");
+        require(_labels.length == numPods, "incorrect labels array");
+        require(_ensStrings.length == numPods, "incorrect ensStrings array");
+        require(_imageUrls.length == numPods, "incorrect imageUrls array");
+
+        uint256 nextPodId = memberToken.getNextAvailablePodId();
+
+        /*
+            When creating multiple pods at the same time, there is a case where one pod may be a dependancy of another
+            createPods -> PodA, PodB, PodC
+            PodB.members[address(0x1337), address(PodA) // doesn't exist yet]
+            since PodA doesn't exist at create time we need a placeholder
+            
+            when deploying the array of pods we use newPods as a cache 
+            as each pod gets deployed we add the address to newPods
+            newPods = [address(0), address(PodA)];
+            PodB.members[address(0x1337), address(1) // we know to check the cache]
+
+            because we can rely on address(0) we have to index the cache at 1
```

Comment:
this should be `because we can't` right?

Reply:
![image](https://user-images.githubusercontent.com/23225162/173862868-8757c045-e8a1-4d00-baab-08a85bc13ed6.png)

Reply:
ack